src/Reference/ContainsOne.php

```diff
@@ -65,7 +65,7 @@ protected function init(): void
                 'reference' => $this,
                 'system' => $this->system,
                 'caption' => $this->caption, // it's ref models caption, but we can use it here for field too
-                'ui' => array_merge_recursive([
+                'ui' => array_replace_recursive([
```

Comment:
see original PR, I think recursive not needed at all

Reply:
See https://3v4l.org/sjJQs for example. That's why we need to merge/replace here recursively.

Form control can have it's own default ui settings (like Multiline control does) and we should be able to extend (not fully overwrite) them.

Reply:
No! here is something totally different, see line 69-70 (that two lines below)

Reply:
see, use `array_replace` when data are hardcoded!

Reply:
ok will go with `array_merge` then.